tide_site.install

```diff
@@ -42,11 +44,20 @@ function tide_site_update_10001() {
     'core.entity_view_display.taxonomy_term.sites.default',
   ];
   foreach ($form_configs as $form_config) {
-    $config = \Drupal::configFactory()->getEditable($form_config);
-    $config_read = _tide_read_config($form_config, $config_location, FALSE);
-    $config->set('dependencies', $config_read['dependencies']);
-    $config->set('content', $config_read['content']);
-    $config->set('hidden', $config_read['hidden']);
-    $config->save();
+    $rewrite = _tide_read_config($form_config, $config_location, FALSE);
+    $display_config_entity = \Drupal::configFactory()->getEditable($form_config);
+    $original_config = $display_config_entity->getRawData();
+    $rewritten_config = NestedArray::mergeDeep($original_config, $rewrite);
+    if ($rewritten_config['dependencies']) {
```

Comment:
The reason I need to handle the dependencies data seems to be due to a bug with `NestedArray::mergeDeep` (I'm not entirely sure), which causes data duplication within dependencies. I might need to check Drupal.org for more information